<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServicesWindow</class>
 <widget class="QMainWindow" name="ServicesWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Пароли от сервисов</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/images/res/icons8-numbers-64.png</normaloff>:/images/res/icons8-numbers-64.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#ServicesWindow{
background: rgb(46, 47, 48);
}

QLineEdit
{
	background: black;
	color:white;
	border: 2px solid white;
	border-radius: 6px;
}

QLabel{
	font-size: 38px;
	font-weight: bold;
	color: solid white;
}

QPushButton{
	
	background: black;
	border: 2px solid white;
	color: white;
	font-size: 12px;
	font-weight: bold;
	border-radius: 6px;
	padding: 20px;
}
QPushButton:hover{
	
	background: white;
	border: 2px solid black;
	color: black;
	border-radius: 6px;
}

QListWidget
{
	border: 2px solid white;
	border-radius: 4px;
	background: rgb(46, 47, 48);
	color: solid white;

}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="servicesListWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>57</y>
      <width>481</width>
      <height>481</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>500</width>
      <height>600</height>
     </size>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="spacing">
     <number>5</number>
    </property>
    <property name="itemAlignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="searchLine">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>10</y>
      <width>381</width>
      <height>40</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>40</number>
    </property>
    <property name="placeholderText">
     <string>Поиск сервиса</string>
    </property>
   </widget>
   <widget class="QPushButton" name="searchButton">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>9</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#searchButton:hover{
	
	icon:url(:/images/res/icons8-search-50.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/images/res/icons8-search-50 (1).png</normaloff>:/images/res/icons8-search-50 (1).png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>27</width>
      <height>26</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="refreshButton">
    <property name="geometry">
     <rect>
      <x>444</x>
      <y>9</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#refreshButton:hover{
	
	icon: url(:/images/res/icons8-refresh-90.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/images/res/icons8-refresh-90 (1).png</normaloff>:/images/res/icons8-refresh-90 (1).png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>27</width>
      <height>26</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteAllButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>544</y>
      <width>481</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#deleteAllButton:hover{
	icon: url(:/images/res/icons8-recycle-bin-64.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/images/res/icons8-recycle-bin-64 (1).png</normaloff>:/images/res/icons8-recycle-bin-64 (1).png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>44</height>
     </size>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
